### PR TITLE
Set resource_name in .tx/config files to preserve resource naming behavior prior to 2.1.0

### LINKED ISSUE #88: Transifex's tx client seems to have changed behavior for resource naming
Previously, I could use the .tx/config file created by `sphinx-intl update-txconfig-resources` to upload resources, whose name would match the resource ID, e.g. "tariffs--1_1--codelists", which comes from normalizing the resource's local file path.

Now, in Transifex, for new resources, the resource shows up with a name that corresponds to the basename of the `source_file`, e.g. `codelists.pot`.

The earlier behavior can be restored by setting `resource_name` in the config file (i.e. adding `--resource-name` to the `tx add` command in `transifex.py`).

If this project doesn't want to hardcode `resource_name` to the resource ID, I would appreciate an opt-in flag for this behavior.

### PR DESCRIPTION
**Pull request by jpmckinney**
closes #88

### CODE REVIEW

File: sphinx_intl/transifex.py [hunk 1]
```diff
@@ -174,6 +175,7 @@ def update_txconfig_resources(transifex_organization_name, transifex_project_nam
     ) as progress_bar:
         for pot_path in progress_bar:
             resource_path = str(pot_path.relative_to(pot_dir).with_suffix(''))
+            resource_slug = normalize_resource_name(resource_path)
             resource_name = normalize_resource_name(resource_path)
```
Comment (m-aciek) [inline sphinx_intl/transifex.py:179]: nitty suggestion
```suggestion
            resource_name = resource_slug = normalize_resource_name(resource_path)
```

**Review by m-aciek — COMMENTED** [review]
Do you mind adding [a test](https://github.com/sphinx-doc/sphinx-intl/blob/master/tests/test_cmd_transifex.py)?

**Comment by jpmckinney** [comment]
@m-aciek I've added a test and updated my commit with the suggestion.

**Review by m-aciek — APPROVED** [review]
Looks good to me. Thanks!

**Comment by jpmckinney** [comment]
I've added an entry :)

**Review by rffontenelle — APPROVED** [review]

**Comment by jpmckinney** [comment]
Is there anything blocking a merge?

**Comment by AA-Turner** [comment]
@jpmckinney tests failed on master, please could you provide a fix? Otherwise we'll need to revert.

A

**Comment by jpmckinney** [comment]
I'll look into it!

**Comment by jpmckinney** [comment]
I'm unable to reproduce and I installed the same versions of packages as listed by pip freeze in the workflow.

**Comment by jpmckinney** [comment]
Aha, something changed in `tx` between 1.6.7 and 1.6.10. Looks like failures start with 1.6.9.

**Comment by jpmckinney** [comment]
Okay, tx just changed the whitespace alignment: https://github.com/sphinx-doc/sphinx-intl/pull/96

**Pull request merged**